sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/hash.scala

```diff
@@ -247,11 +247,8 @@ abstract class HashExpression[E] extends Expression {
   override def nullable: Boolean = false
 
   override def checkInputDataTypes(): TypeCheckResult = {
-    if (children.isEmpty) {
-      TypeCheckResult.TypeCheckFailure("function hash requires at least one argument")
-    } else {
-      TypeCheckResult.TypeCheckSuccess
-    }
+    TypeUtils.checkTypeInputDimension(
+      children.map(_.dataType), s"function $prettyName", requiredMinDimension = 1)
```

Comment:
Oh, now we have a consistent message instsead of `at least one argument` or `cannot be empty`. :)